packages/mobile/src/recipients/RecipientItem.tsx

```diff
@@ -24,55 +26,50 @@ class RecipientItem extends React.PureComponent<Props> {
     const { recipient } = this.props
 
     return (
-      <TouchableHighlight onPress={this.onPress} underlayColor={colors.altDarkBg}>
-        <View style={style.row}>
+      <Touchable onPress={this.onPress}>
```

Comment:
Nothing to do now but I think we should have a `TouchableItem` component to use in scrollable list of items that would use `TouchableHighlight` on iOS and the something based on `TouchableNativeFeedback` on Android, to respect platform conventions.

`Touchable` alone currently uses `TouchableOpacity` on iOS and `TouchableNativeFeedback` on Android.